docs/samples/llmisvc/dp-ep/llm-inference-service-dp-ep-qwen-gpu.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: LLMInferenceService
+metadata:
+  annotations:
+    leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology: kubernetes.io/hostname
+  name: qwen3-30b-a3b-fp8
+  namespace: default
+spec:
+  model:
+    name: Qwen/Qwen3-30B-A3B-FP8
+    uri: hf://Qwen/Qwen3-30B-A3B-FP8
+  parallelism:
+    data: 2
+    dataLocal: 1
+    expert: true
+    tensor: 1
+  router:
+    gateway: { }
+    route: { }
+    scheduler: { }
+  template:
+    hostIPC: true
+    hostPID: true
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: nvidia.com/gpu.product
+                  operator: In
+                  values:
+                    - NVIDIA-H100-80GB-HBM3
```

Comment:
Do we need to enforce it?

Reply:
This example is outdated, so I'll drop it